apps/www/src/lib/registry/default/ui/toast/Toast.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import { ToastRoot, type ToastRootEmits, type ToastRootProps, useEmitAsProps } from 'radix-vue'
+
+import { toastVariants } from '.'
+import { cn } from '@/lib/utils'
+
+export interface ToastProps extends ToastRootProps {
+  class?: string
+  variant?: NonNullable<Parameters<typeof toastVariants>[0]>['variant']
+};
```

Comment:
Thank you for PR

If you look into main `radix-vue` source, they are using normal script alongside script setup for exporting types

```suggestion
<script lang="ts">
import type { VariantProps } from 'class-variance-authority'

export interface ToastProps extends ToastRootProps, VariantProps<typeof toastVariants> {
  class?: string
};
</script>
<script setup lang="ts">
```